### ISSUE TITLE
Fakegato history

### ISSUE THREAD

**Issue by craigdent** [kobim/homebridge-efergy-ego#1]
Awesome work on the plugin!

Any chance of adding in fakegato history?

**Comment by kobim** [comment]
Hi @craigdent, thank you!

I'm not familiar with _fakegato history_, could you please elaborate on that?

**Comment by craigdent** [comment]
Oh really? Sorry just assumed haha. 

Here is the link on it... it’s basically the same principal as you see in EVE in the sense you get a history of the device usage... should be fairly easy to incorporate into your plugin if your keen too, be awesome if you did :)

https://github.com/simont77/fakegato-history

**Comment by kobim** [comment]
I’m not sure how this accessory relates to Eve app or the Eve’s products line at all.

The usage history is written at the device and you should be able to see it on the Efergy EGO app as usual.

**Comment by craigdent** [comment]
Yes.. but you integrate the fakegato history into your plugin so when you open EVE app it displays history

Like the homebridge-efergy plugin did

https://www.npmjs.com/package/homebridge-efergy-mg
